docs/examples/viz_helical_motion.py

```diff
@@ -35,21 +34,25 @@
 #   timer_callback (default = 0.09)
 # * `angular_frq`: angular frequency (default = 0.1)
 # * `phase_angle`: phase angle (default = 0.002)
+# * `radius_path`: radius of the path followed by the particle (default = 1.5)
 #
 
 radius_particle = 0.08
 initial_velocity = 0.09
 acc = 0.004
 time = 0
 incre_time = 0.09
-angular_frq = 0.1
+angular_frq = 0.11
 phase_angle = 0.002
+radius_path = 1.5
 
 
 ###############################################################################
 # Creating a scene object and configuring the camera's position
 
 scene = window.Scene()
+scene.background((1, 0, 0))
+scene.GradientBackgroundOn()
```

Comment:
@SunTzunami Add extra options in ``scene.background`` to enable the gradient. Do not call a vtk method here like that. Thx.

Reply:
Done.